.changeset/config.json

```
@@ -6,6 +6,6 @@
   "fixed": [],
   "linked": [],
   "access": "public",
-  "updateInternalDependencies": "patch",
+  "updateInternalDependencies": "minor",
```

I think you've changed the wrong patch

```suggestion
  "updateInternalDependencies": "patch",
```

---

updated